src/Impostor.Server/Net/Inner/InnerNetObject.Anticheat.cs

```diff
@@ -73,9 +73,16 @@ protected async ValueTask<bool> ValidateCmd(CheatContext context, IClientPlayer
             return true;
         }
 
-        protected async ValueTask<bool> ValidateImpostor(CheatContext context, IClientPlayer sender, InnerPlayerInfo playerInfo, bool value = true)
+        protected async ValueTask<bool> ValidateImpostor(CheatContext context, IClientPlayer sender, InnerPlayerInfo? playerInfo, bool value = true)
         {
-            if (playerInfo.IsImpostor != value)
+            if (playerInfo == null)
+            {
+                if (await sender.Client.ReportCheatAsync(context, CheatCategory.InvalidObject, "Couldn't check if Impostor, playerInfo not set"))
```

Comment:
MINOR: if the invalid objects category is off but role checks is not, people are able to murder with an invalid object. Either:
1. Check both categories
2. Check Role here instead of InvObj
3. Severely warn against turning off InvObj and accept cases like this (aka documentation update)

Idem Validate{CanVent,Role}

Reply:
Documentation update was chosen